signs-work-order-map/components/SignPopup.tsx

```diff
@@ -0,0 +1,40 @@
+import { Popup } from "react-map-gl/mapbox";
+import { Sign } from "@/types/map";
+
+interface SignPopupProps {
+  popupInfo: Sign;
+  setPopupInfo: React.Dispatch<React.SetStateAction<Sign | null>>;
+}
+
+export default function SignPopup({ popupInfo, setPopupInfo }: SignPopupProps) {
+  // prod has the font as bold, do we keep?
+  return (
+    <Popup
+      anchor="top"
+      longitude={popupInfo.lng}
+      latitude={popupInfo.lat}
+      onClose={() => setPopupInfo(null)}
+      offset={[0, -20]}
+    >
+      <ul className="list-unstyled m-0">
+        {!popupInfo.locationDetailPage && (
+          <li>
+            <a
+            // TODO: update to prod url when testing is completed
```

Comment:
maybe it would be beneficial to set this url in a variable somewhere? idk how many times the Knack app link is used around the app but could help with future development